### DISCUSSION TITLE
image is shown blurred for 2s, then dissapears and gives 404 after a while

### DISCUSSION THREAD

**Discussion by lotje-kinable** [spatie/laravel-medialibrary#2000] [Q&A]
Hi

Laravel7, Ubuntu 18.04.5, php 7.4

On my local machine all works well, responsive images/conversions... however I cloned my repo to a Digital Ocean droplet and I used the command 'php artisan storage:link' and the images are shown blurred for about 2s and then I get a 404 on the link to the image.  I disabled 'responsive images' and later 'conversions' and I can't even get the normal image (I deleted all db entries and started fresh after disabling them.) 

when I run php artisan storage:link, I get: 
The [/var/www/htmls/public/storage] link has been created 

So maybe this is wrong? Or do I need to configure the sites-available/default differently... ? 

I'm a little lost atm, I don't know where to start or what to include, I was hoping someone might recognise this and point out something I just missed... otherwise, please ask which files you would like to see to be able to help me out! 

This is the first time I post something anywhere, so please point out any mistakes/corrections! 

Thanks in advance! 

ps. I've only been programming in Laravel for 6 months :-) 
![Screenshot 2020-08-16 at 10 39 56](https://user-images.githubusercontent.com/17978479/90330415-d6d6ed00-dfac-11ea-8fe6-6b0046be93bd.png)

**Comment by francoism90** [comment]
The media files aren't public accessable and do not live in the public folder, you need to use the getUrl methods provided by this packages.
